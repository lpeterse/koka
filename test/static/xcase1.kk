import static/modA 
alias bar = modA/aint
