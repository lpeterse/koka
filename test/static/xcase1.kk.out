static/xcase1/bar: V
public alias bar = static/modA/aint = 2